test/e2e/core.go

```diff
@@ -62,6 +63,8 @@ func CoreDump(dir string) {
 		cmds = append(cmds, []command{
 			{"cat /var/log/kubelet.log", "kubelet"},
 			{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+			{"cat /var/log/supervisor/kubelet-stdout.log", "supervisord-kubelet-stdout"},
+			{"cat /var/log/supervisor/kubelet-stderr.log", "supervisord-kubelet-stderr"},
```

Comment:
ah, forgot to pick up docker.log. Will add them. I couldn't make docker-stdout/stderr.log have more information. All I can get is 

```
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```